### DISCUSSION TITLE
v11 Beta 1

### DISCUSSION THREAD

**Discussion by joshblack** [carbon-design-system/carbon#9376] [Release Feedback]
Have you tried out the v11 Beta 1 release and have any feedback? We'd love to hear it! From issues that you might have run into to questions you might have, feel free to post anything release-related here. 

Thanks so much for your help! We really appreciate it 🙏

**Comment by adrianamorales** [comment]
@joshblack 

**Feedback on tokens (as a designer):**
I've been playing around with the white beta kit and the new color token names are so much easier for my brain to process! Especially working with so many nested elements, I can better understand and see the hierarchy of elements in the page.

**Color implementation:** 
The breakdown between layer tokens, contextual layer tokens, and inline theming is super helpful. As a power user of Carbon and themes, seeing the tokens in action in the images help. 

In Cloud PAL, we tried to explain these concepts to teams but people got lost in the details. Although the docs are helpful, I wonder if there's a TL;DR version or a cheatsheet that Carbon or a PAL could create to make the learning curve less daunting. 

**Illustration guidance:** 
As I mentioned in slack, Cloud PAL can contribute to this guidance. :) 

**Developing for light or dark mode:** 
Cloud PAL can contribute guidance on inline theming and using tokens. :)

**Comment by mattrosno** [comment]
I have `@carbon/react@0.2.0` installed and have uninstalled `carbon-components` and `carbon-components-react`. I'm getting this error:

```
Attempted import error: 'unstable_ContextMenu' is not exported from 'carbon-components-react'.
```

Not seeing the source anywhere...

https://github.com/carbon-design-system/carbon/search?q=unstable_ContextMenu

Related to this?

https://github.com/carbon-design-system/carbon/issues/9383

@joshblack @tw15egan

**Reply by jnm2377** [reply]
For anyone else running into this issue, this was fixed in this PR: https://github.com/carbon-design-system/carbon/pull/9476 and has been released under the `next` tag for `@carbon/react` for now: https://github.com/carbon-design-system/carbon/pull/9490

**Comment by abbeyhrt** [comment]
This is a super small nitpick but in the migration guide [here](https://v11.carbondesignsystem.com/migrating/guide/develop#how-to-migrate) it suggests `yarn install @carbon/react` instead of `yarn add @carbon/react` for the command line installation instructions. Unless that's changed! I'm still on v1 of yarn so it might have gone back to `install`.

**Comment by joshblack** [comment]
Something else that's small that we need to figure out an interim solution for:

`@carbon/react` depends on `carbon-components-react` which has peer dependencies. We should install those in the interim so people don't have to do it themselves.

**Reply by jnm2377** [reply]
To add to that, when we were migrating the Carbon Tutorial, we had to install `carbon-components` and `carbon-icons` as peer dependencies. It wouldn't build without those. However, our imports were all pointing to `@carbon/react`.

**Comment by joshblack** [comment]
Seems like we might need to make sure there are storybook stories available / linked for:

- Inline theming
- Light prop alternative
- CSS Grid

We also need to make sure it’s obvious where/how to do things using our Sass package, ideally linking our sass docs

**Comment by joshblack** [comment]
Migration guide / post ideas:

- How to migrate from Flexbox grid to CSS Grid implementation
- How to change tokens or find tokens to change
- How to implement inline theming (in particular how to move away from experimental work)
- How to implement light mode / dark mode
- How to use the compat theme

**Comment by joshblack** [comment]
Nice example post from the SWR 1.0 release: https://swr.vercel.app/blog/swr-v1

Love their sidebar

**Comment by abbeyhrt** [comment]
Would it be possible for the list-box styles to also be exported through `@carbon/react/scss/components/`?
